### PR TITLE
Debug render warnings, delete console.logs

### PR DESCRIPTION
**Pull request by kokosaka**

### CODE REVIEW

**Review by riz671 — APPROVED** [review]

**Comment by kokosaka** [comment]
Simulator Screen Shot - iPhone 11 Pro - 2020-11-10 at 16 32 18
Simulator Screen Shot - iPhone 11 Pro - 2020-11-10 at 16 32 15

File: screens/ProfileScreen.js [hunk 1]
```diff
@@ -151,6 +152,12 @@ const ProfileScreen = (props) => {
     // { label: "Non-Binary", value: "non-binary" },
   ];
 
+  const goalList = [
```
Comment (riz671) [inline screens/ProfileScreen.js:155]: Bloody Brilliant

File: screens/HomeScreen.js [hunk 2]
```diff
@@ -82,6 +94,48 @@ const HomeScreen = (props) => {
...
+  const getCalIntake = () => {
+    let yr = moment().format("YYYY");
+    let mm = moment().format("MM");
+    let dd = moment().format("D");
+    firebase
+    .database()
+    .ref(`users/${props.displayName}/foodJournal/${yr}/${mm}/${dd}`)
+    .on("value", function(snapshot) {
+      let calories = 0;
+      Object.values(snapshot.val()).map((recipe) => {
```
Comment (riz671) [inline screens/HomeScreen.js]: Neat!

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -53,6 +59,12 @@ const HomeScreen = (props) => {
     }
   }, [props.profilePic]);
 
+  useEffect(() => {
+    getCalExpend();
+    getCalIntake();
+    getCalGoal();
+  })
```
Comment (riz671) [inline screens/HomeScreen.js]: This use effect is unoptimized. Right now, the useEffect runs EVERYTIME an update happens, which can be taxing to the system by causing muliple re-renders if something changes in homescreen. I change profile picture, BOOM, use effect runs again. I click log out, state changes, BOOM, use effect runs again. What we want is to update once user is retrieved.

You fix that is by adding an array of values as the second argument of useEffect such that we don't re-render page unneccesarily. add [props.user] as second argument of this useEffect. You should be good!

**Review by riz671 — CHANGES_REQUESTED** [review]
Fix the use Effect and you should be good!

**Review by riz671 — APPROVED** [review]
Looking good. Way to go!

**Pull request merged**